### ISSUE TITLE
Performance data/units

### ISSUE THREAD

**Issue by taam** [nbuchwitz/check_pve#30]
There are some issues regarding performance data and units, in particular `MB` is used with `2^20` as conversion factor, which is wrong (in terms of SI units), the correct unit for this conversion factor would be `MiB`. If `MB` is kept, the correct factor of `10^6` should be used.

Performance data currently is technically wrong for the same reason. However, consumers treat `MB` differently, e.g. Icinga before version 2.13.0 treats `MB` like `MiB`, using `2^20` as conversion factor, so it kinda works despite being not really correct. Icinga since version 2.13.0 learned about new units and should now treat them correctly, so `MiB` could be used, but there are probably other consumers that can't handle `MiB`. (Also Icingaweb2 seems to do it's own unit/conversion handling and currently messes things up as well.) So the best solution regarding performance data might be to not emit a scaled unit but just bytes with `B`.

**Comment by taam** [comment]
There's another issue with the performance data I forgot to mention: The `max` value is put in the `min` column, I suggest adding a min value where appropriate (e.g. `0` for memory/swap/storage).

**Comment by nbuchwitz** [comment]
Good catch! I created a first draft of a solution here: https://github.com/nbuchwitz/check_pve/tree/fix/performance-data-units.
This way we keep the existing numbers (not the unit) as default and allow the user for custom units with the option `--unit``

Looking forward for your feedback!

**Comment by bgabika** [comment]
Dear Nicolai,

Thank you for your wonderful check plugin for Proxmox.
I use latest version of check_pve.py: Version: 1.2.2

I faced an issue with warning and critical values of storage checking.
Help says:

 -w THRESHOLD_WARNING, --warning THRESHOLD_WARNING
                        Warning threshold for check value. Mutiple thresholds with name:value,name:value
 -c THRESHOLD_CRITICAL, --critical THRESHOLD_CRITICAL
                        Critical threshold for check value Mutiple thresholds with name:value,name:value


When I run check command without name:value,name:value threshold formula, I get WARNING and CRITICAL exit codes:

	# ./check_pve.py -u monitoring@pve -p nagios -e 172.26.73.22 -k -m storage -n pve2 --name ceph_hdd -w 20 -c 25 --unit GB
	CRITICAL - Usage of storage 'ceph_hdd' is 74.79 %|usage=74.79%;20.0;25.0;0; used=12568.69GB;20.0;25.0;0;16804.76

But when I run check command with name:value,name:value thresholds, usage warning and critical thresholds do not working:

	# ./check_pve.py -u monitoring@pve -p nagios -e 172.26.73.22 -k -m storage -n pve2 --name ceph_hdd -w usage:20,used:0 -c usage:25,used:0 --unit GB
	OK - Usage of storage 'ceph_hdd' is 74.87 %|usage=74.87%;20.0;25.0;0; used=12578.57GB;0.0;0.0;0;16799.66


What can be wrong?

thank you,
Gabor
